crates/integration_tests/tests/shared/append_partition_data_file_test.rs

```diff
@@ -77,13 +78,12 @@ async fn test_append_partition_data_file() {
         .expect("could not bind to schema");
 
     let table_creation = TableCreation::builder()
-        .name("t1".to_string())
+        .name("t2".to_string())
```

Comment:
Making the write tests use different tables each allows for them to also use the shared docker stack.